pkg/apis/core/validation/validation_test.go

```diff
@@ -9952,159 +9909,149 @@ func TestValidatePodSpec(t *testing.T) {
 	badfsGroupChangePolicy2 := core.PodFSGroupChangePolicy("")
 
 	successCases := map[string]core.PodSpec{
```

Comment:
As a small simplification, if this was an array of `Pod`, rather than `PodSpec`, you would not need to add `.Spec` on each testcase.  Just extract the Spec field in the test loops which consume these arrays.